lib/kamal/secrets/adapters.rb

```diff
@@ -3,6 +3,7 @@ module Kamal::Secrets::Adapters
   def self.lookup(name)
     name = "one_password" if name.downcase == "1password"
     name = "last_pass" if name.downcase == "lastpass"
+    name = "gcp_secret_manager" if %w[gcp secret_manager].include? name.downcase
```

Comment:
Agree with the `gcp` alias but not sure if `secret_manager` is needed and also maybe too generic?

Reply:
Thanks for the review, that makes sense :+1: